app/views/admin/cfps/index.html.haml

```diff
@@ -0,0 +1,36 @@
+.row
+  .col-md-12
+    .page-header
+      %h1 Call for Papers
+      %p.text-muted
+        Call for people to submit events to your conference
+- if @program.cfps
+  .row
+    .col-md-12
+      %table.table.table-hover.datatable#tickets
+        %thead
+          %th Type
+          %th Start Date
+          %th End Date
+          %th Days Left
+          %th Actions
+        %tbody
+          - @program.cfps.each do |cfp|
+            %tr
+              %td
+                = link_to(admin_conference_program_cfp_path(@conference.short_title, cfp.id)) do
+                  = cfp.cfp_type.capitalize
+              %td
+                = cfp.start_date.strftime('%A, %B %-d. %Y')
+              %td
+                = cfp.end_date.strftime('%A, %B %-d. %Y')
+              %td
+                = pluralize(cfp.remaining_days, 'day')
+              %td
+                .btn-group
+                  = link_to 'Edit', edit_admin_conference_program_cfp_path(@conference.short_title, cfp.id), method: :get, class: 'btn btn-primary'
+                  = link_to 'Delete', admin_conference_program_cfp_path(@conference.short_title, cfp.id), method: 'delete', class: 'btn btn-danger', data: { confirm: 'Are you sure you want to delete the CfP?' }
+- if @program.remaining_cfp_types.length > 0
```

Comment:
Add an ability so that:
1) This line becomes something like i`f can? [:new, :create] @program.cfps.new` (alternatively you could still be showing the button, just make it disabled with a title saying that it is not possible to create any other type of CfP for the conference
2) You also authorize the access to the action new/create so that with load_and_authorize in controller it will not be possible to access this page at all